code/game/objects/configuration.dm

```diff
@@ -0,0 +1,26 @@
+/obj/configuration
+	name = "map configuration"
+	desc = "This is a pseudo-object used for configuring various subsystems for specific maps."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "blueprints"
+	anchored = TRUE
+	density = TRUE
+
+/obj/configuration/Initialize(mapload)
+	. = INITIALIZE_HINT_QDEL
+
+/obj/configuration/shuttle_loader
+	name = "shuttle loader configuration"
+	desc = "Change `shuttle_id` var to selected shuttle to load it for this map."
+	var/shuttle_id
+
+/obj/configuration/shuttle_loader/Initialize(mapload)
+	. = ..()
+	for(var/id in SSmapping.shuttle_templates)
+		var/datum/map_template/shuttle/D = SSmapping.shuttle_templates[id]
+		if(D.shuttle_id == shuttle_id)

```

Comment:
shuttle templates is an associative array with the keys being id. Get your map template by doing
`var/datum/map_template/shuttle/D = SSmapping.shuttle_templates[shuttle_id]`